src/elements/dv-elements/utils/ajax-ls/view-file.html

```diff
@@ -338,6 +343,13 @@
                 }
                 this._updateContentElevationAttribute();
             }
+            _fileNameCompare(a, b) {
+                if (a.fileName < b.fileName)
```

Comment:
may be: `return a.localeCompare(b)`  ?

Reply:
Quite possibly, I'm by no means a JS expert so feel free to clean up any C-isms before merging :)